<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="cupAV">
    <link name="link">
      <pose>0 0 0 0 0 0</pose>
      <inertial>
        <mass>0.5</mass>
        <inertia>
          <ixx>0.0121460888</ixx>
          <ixy>0.00041119425</ixy>
          <ixz>-0.00017330985</ixz>
          <iyy>0.0134317552</iyy>
          <iyz>5.7374125e-05</iyz>
          <izz>0.0137008272</izz>
        </inertia>
      </inertial>

      <collision name="collision">
        <geometry>
          <mesh>
            <uri>model://cupAV/meshes/whiteCup.dae</uri>
            <scale>0.001 0.001 0.001</scale>
          </mesh>
        </geometry>
        <surface>
          <contact>
            <ode>
              <max_vel>0.1</max_vel>
              <min_depth>0.001</min_depth>
              <soft_erp>0.5</soft_erp>
              <soft_cfm>0.001</soft_cfm>
            </ode>
          </contact>
          <friction>
            <ode>
              <mu>10000.0</mu>
              <mu2>10000.0</mu2>
            </ode>
          </friction>
        </surface>
      </collision>

      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://cupAV/meshes/whiteCup.dae</uri>
            <scale>0.001 0.001 0.001</scale>
          </mesh>
        </geometry>
        <material>
          <ambient>1 0 0 1</ambient>
          <diffuse>1 0 0 1</diffuse>
          <specular>0.1 0.1 0.1 1</specular>
          <emissive>0 0 0 0</emissive>
        </material>
      </visual>
    </link>
  </model>
</sdf>
